programs/svm-spoke/src/instructions/admin.rs

```diff
@@ -257,17 +259,21 @@ pub fn relay_root_bundle(
 #[instruction(root_bundle_id: u32)]
 pub struct EmergencyDeleteRootBundle<'info> {
     #[account(
-        mut,
         constraint = is_local_or_remote_owner(&signer, &state) @ CustomError::NotOwner
     )]
     pub signer: Signer<'info>,
 
+    #[account(mut)]
+    // We do not restrict who can receive lamports from closing root_bundle account as that would require storing the
```

Comment:
makes sense, as long as it's authenticated with `is_local_or_remote_owner` we don't really mind.

Reply:
In remote authentication this checks only that this is coming from HubPool via CCTP, but we do not use permissioned caller to receive the message on Solana side (though CCTP has that option). So there might be a case that someone frontruns receiving message on Solana and gets all lamports from the closing of root bundle account. But I think this is rare enough when we would use this emergency method and it should be no concern that someone else could gain some extra lamports.

Reply:
Makes sense. Anyone can "finalize" this messages